nix/tools.nix

```diff
@@ -24,6 +24,7 @@
 , html-tidy
 , hunspell
 , luaPackages
+, sumneko-lua-language-server
```

Comment:
Note that this package has been renamed to `lua-language-server` in `nixpkgs-unstable`.